packages/devtools_app/lib/src/shared/charts/chart.dart

```diff
@@ -524,36 +525,22 @@ class ChartPainter extends CustomPainter {
     return label == 0 ? '0' : '$label$unit';
   }
 
-  void drawXTick(
-    Canvas canvas,
-    int timestamp,
-    double xTickCoord,
-    Paint axis, {
-    bool shortTick = true,
-    bool displayTime = false,
-  }) {
-    if (displayTime) {
-      // Draw vertical tick (short or long).
-      canvas.drawLine(
-        Offset(xTickCoord, 0),
-        Offset(xTickCoord, shortTick ? 2 : 6),
-        axis,
-      );
```

Comment:
why did we remove this?

Reply:
`displayTime` is never passed as `true` (an argument is never given to the `drawXTick` calls). This was revealed by making the method private.